src/styles/styles.js

```diff
@@ -1017,6 +1017,22 @@ const styles = {
         alignItems: 'center',
         padding: 20,
     },
+
+    subHeader: {
+        color: themeColors.text,
+        fontWeight: fontWeightBold,
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: 12,
+        padding: 20,
+    },
+
+    selectCircle: {
+        width: 30,
+        height: 30,
+        borderColor: colors.charcoal,
```

Comment:
Can we use `themeColors.border` please?

Reply:
Sure thing, it does look a little light to me